english/marshmallow.md

```diff
@@ -781,7 +781,7 @@ When I code over my weekend, I feel that this becomes my [whole life](code-my-li
 Suddenly you feel a strong craving for [something.](craving/craving.md)
 
 Despite the little conundrum that you resolved, you realize that it's now 
-Friday, December 4th.  
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.